### ISSUE TITLE
Invalid glTF from exporting armatures linked across scenes

### ISSUE THREAD

**Issue by Capital-Asterisk** [KhronosGroup/glTF-Blender-IO#1576]
**Describe the bug**
An armature linked across multiple scenes produces an invalid glTF file.

This invalid gltf cannot be re-imported back into blender, and causes a python error. A gltf validator will produce a "NODE_PARENT_OVERRIDE" error.

This is caused by multiple exported copies of the armature nodes sharing the same bone nodes as children:

```json
"nodes" : [
    {
        "name" : "Bone"
    },
    {
        "children" : [
            0
        ],
        "name" : "Armature"
    },
    {
        "children" : [
            0
        ],
        "name" : "Armature"
    }
]
```

**To Reproduce**
1. Open a default file and delete everything
2. Create an armature
4. Create a new scene, a Linked Copy
5. Export a gltf file, any file format works
6. Import the new gltf file back into blender, it should produce this error:
```
Python: Traceback (most recent call last):
  File "/usr/share/blender/3.0/scripts/addons/io_scene_gltf2/__init__.py", line 1064, in execute
    return self.import_gltf2(context)
  File "/usr/share/blender/3.0/scripts/addons/io_scene_gltf2/__init__.py", line 1078, in import_gltf2
    if self.unit_import(path, import_settings) == {'FINISHED'}:
  File "/usr/share/blender/3.0/scripts/addons/io_scene_gltf2/__init__.py", line 1098, in unit_import
    BlenderGlTF.create(gltf_importer)
  File "/usr/share/blender/3.0/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_gltf.py", line 42, in create
    BlenderGlTF._create(gltf)
  File "/usr/share/blender/3.0/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_gltf.py", line 49, in _create
    BlenderScene.create(gltf)
  File "/usr/share/blender/3.0/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_scene.py", line 42, in create
    compute_vnodes(gltf)
  File "/usr/share/blender/3.0/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_vnode.py", line 26, in compute_vnodes
    init_vnodes(gltf)
  File "/usr/share/blender/3.0/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_vnode.py", line 129, in init_vnodes
    assert gltf.vnodes[child].parent is None
AssertionError

location: <unknown location>:-1
```

**Expected behavior**
Exporting invalid gltf shouldn't be possible. When armatures are linked within the same scene, then the bones are duplicated too:

```json
"nodes" : [
    {
        "name" : "Bone"
    },
    {
        "children" : [
            0
        ],
        "name" : "Armature"
    },
    {
        "name" : "Bone"
    },
    {
        "children" : [
            2
        ],
        "name" : "Armature.001"
    }
]
```

I believe this behaviour should also be the case for an armature linked across two scenes.

**Version**
 - OS: linux
 - Blender Version: 3.0.1 [dc2d18018171]

**Comment by Capital-Asterisk** [comment]
I've just checked out #1378, and I can confirm this issue no longer happens in the new changes.

**Comment by julienduroure** [comment]
Thanks. 
#1378 should be merged soon.

**Comment by julienduroure** [comment]
Fixed by 69746b2b31eeee080d5dfd3c74f1e02742795ed2
